src/shared/components/MSKTabs/MSKTabs.tsx

```diff
@@ -319,7 +326,14 @@ export class MSKTabs extends React.Component<IMSKTabsProps, IMSKTabsState> {
                     ref={this.tabRefHandler.bind(this, tab.props.id)}
                     className={activeClass}
                 >
-                    <a className={classnames("tabAnchor", `tabAnchor_${tab.props.id}`, tab.props.anchorClassName)} onClick={this.setActiveTab.bind(this,tab.props.id, tab.props.datum)} style={tab.props.anchorStyle}>{tab.props.linkText}</a>
+                    <a
+                        className={classnames("tabAnchor", `tabAnchor_${tab.props.id}`, tab.props.anchorClassName)}
+                        onClick={this.tabClickHandlers(tab.props)}
+                        href={this.props.getTabHref && this.props.getTabHref(tab.props.id)}
```

Comment:
interesting that if you pass undefined to attribute, React knows to just leave attribute off. I test it and it seems to work.